packages/tocco-ui/src/FormattedValue/FormattedValue.stories.js

```diff
@@ -1,30 +1,235 @@
+    () => <FormattedValue type="document-compact" value={{
+      alt: text('alt', 'modern building captured from frog view'),
+      binaryLink: text('binaryLink', 'https://picsum.photos/1000/1000?image=1081'),
+      caption: text('caption', 'modern skyscraper'),
+      fileName: text('fileName', 'modern_skyscraper.jpg'),
+      thumbnailLink: text('thumbnailLink', 'https://picsum.photos/400/400?image=1081')
+    }}/>,
+    {info: {propTables: [DocumentCompactFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Duration',
+    () => <FormattedValue type="duration" value={number('Duration in ms', 83456678)}/>,
+    {info: {propTables: [DurationFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'HTML',
+    () => <FormattedValue type="html" onChange={() => {}} value={text('HTML', storybookHtmlMarkup)}/>,
+    {info: {propTables: [HtmlFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Login',
+    () => <FormattedValue type="login" value={{username: text('username', 'tocco_user')}}/>,
+    {info: {propTables: [LoginFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Longitude',
+    () => <FormattedValue type="longitude" value={{value: number('Longitude', 45.976575)}}/>,
+    {info: {propTables: [LongitudeFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Latitude',
+    () => <FormattedValue type="latitude" value={{value: number('Latitude', 7.658452)}}/>,
+    {info: {propTables: [LongitudeFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Money',
+    () => <FormattedValue type="moneyamount" value={number('Money', 1235.67)}/>,
+    {info: {propTables: [MoneyFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Multi Remote',
+    () => <FormattedValue
+      type="multi-remote"
+      value={object(
+        'Multi Remote',
+        [{key: '1', display: 'apple'}, {key: '2', display: 'khaki'}]
+      )}
+    />,
+    {info: {propTables: [MultiSelectFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Multi Select',
+    () => <FormattedValue
+      type="multi-select"
+      value={object(
+        'Multi Select',
+        [
+          {key: '3', display: 'Matterhorn'},
+          {key: '4', display: 'Jungfraujoch'},
+          {key: '5', display: 'Rigi'}
+        ]
+      )}
+    />,
+    {info: {propTables: [MultiSelectFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Percent',
+    () => <FormattedValue type="percent" value={number('Percent', 65.89)}/>,
+    {info: {propTables: [PercentFormatter], propTablesExclude: [FormattedValue], source: false}}
+  )
+  .add(
+    'Phone',
+    () => <FormattedValue type="phone" value={text('Phone', '+41761234567')}/>,
+    {info: {propTables: [PhoneFormatter], propTablesExclude: [FormattedValue], source: false}}
```

Comment:
I think it's problematic to show Props of PhoneFormatter here since PhoneFormatter is a hidden component that is never used directly. why not simple show propTable and Source of Formatted. So basically delete all these info configs. this does also  make the story cleaner (less code)